### PR TITLE
Update grunt and grunt tools

### PR DESCRIPTION
**Pull request by elesdoar**
- Update grunt
- Update grunt tasks
- Refactor change opacity method.
- Update website

### CODE REVIEW

**Comment by elesdoar** [comment]
@nmccready I have updated Grunt, do you agree?

File: bower.json [hunk 1]
```diff
@@ -34,7 +38,7 @@
     "Leaflet.utfgrid": "danzel/Leaflet.utfgrid",
     "Leaflet.awesome-markers": "*",
     "leaflet-providers": "~1.1.5",
-    "leaflet.vector-markers": "~0.0.3",
+    "leaflet.vector-markers": "0.0.4",
```
Comment (nmccready) [inline bower.json]: hah, i just made this change on 1.X

File: package.json [hunk 2]
```diff
@@ -74,5 +75,8 @@
     "angular": "1.x",
     "angular-simple-logger": "0.1.x",
     "leaflet": "0.7.x"
+  },
+  "engines": {
+    "npm": ">=3.0.0"
```
Comment (nmccready) [inline package.json]: Actually can we not have this restriction. I am still on npm 2 and npm 3 has been nothing but slow for me.

File: bower.json (hunk 1 above)
Comment (elesdoar) [inline bower.json]: What is the branch diference? 1.X vs master, I don't remember it.

File: package.json (hunk 2 above)
Comment (elesdoar) [inline package.json]: OK, I'll remove this.

**Comment by nmccready** [comment]
yeah updating grunt should be fine

**Comment by nmccready** [comment]
Actually be careful with updating grunt-bump though as it will break our bump flow. Unless you want to fix those problems.

**Comment by nmccready** [comment]
eh NVM, https://github.com/vojtajina/grunt-bump/blob/master/package.json#L74 seems to be the same grunt-conventional-changelog

grunt-bump should be fine. I like that they added dry mode.

**Comment by nmccready** [comment]
LGTM

**Pull request merged**